### PR TITLE
Jit64: Make psq_lXX PIE-compliant

### PR DESCRIPTION
**Pull request by MerryMage**
Split the `pairedLoadQuantized` array into two, so the quantized load code matches the quantized store code.

Exact same method as with quantized stores.

### CODE REVIEW

**Comment by degasus** [comment]
Sounds fine

File: Source/Core/Core/PowerPC/JitCommon/JitAsmCommon.h [hunk 1]
```diff
@@ -40,6 +40,12 @@ class CommonAsmRoutinesBase
   const u8** pairedLoadQuantized;
 
   // In: array index: GQR to use.
+  // In: ECX: Address to read from.
+  // Out: XMM0: Bottom 32-bit slot holds the read value.
+  // Trashes: all three RSCRATCH
+  const u8** singleLoadQuantized;
```
Comment (BhaaLseN) [inline Source/Core/Core/PowerPC/JitCommon/JitAsmCommon.h]: Not too sure, but is this a place where `std::array` could help us get rid of some magic constants (such as the `8` in `8 * sizeof(u8*)` for `ReserveCodeSpace` inside `CommonAsmRoutines::GenQuantizedXxxLoads`)?

Then again, this would probably also apply to the paired ones above, so it could be out of scope here and should be its own PR...

Comment (MerryMage) [inline Source/Core/Core/PowerPC/JitCommon/JitAsmCommon.h]: You *could* replace it by `std::array<const u8*, 8>* singleLoadQuantized;`, though the only place that needs the size is the initialization code.

Comment (BhaaLseN) [inline Source/Core/Core/PowerPC/JitCommon/JitAsmCommon.h]: Was just a thought/suggestion. A clean-up PR at a later point is fine by me too.
